docs/facility_e4s.md

```diff
@@ -13,4 +13,4 @@ Summit | ORNL | 21.08 | 2025 | `gcc@8.3.1`, `gcc@9.1.0`, `gcc@9.3.0`, `gcc@10.2.
 Crusher | ORNL | 21.08 | 1426 | `gcc@7.5.0`, `gcc@9.3.0`, `gcc@10.3.0`, `gcc@11.2.0`, `cce@13.0.0`, `cce@13.0.1`, `clang@13.0.0-rocm4.5.0` | [0add25a508f7b78a5a7ea5a6129388a6ed29a993](https://github.com/mpbelhorn/olcf-spack/commit/0add25a508f7b78a5a7ea5a6129388a6ed29a993) | [https://github.com/spack/spack-configs/blob/main/OLCF/crusher/spack.yaml](https://github.com/spack/spack-configs/blob/main/OLCF/crusher/spack.yaml) | [https://github.com/spack/spack-configs/blob/main/OLCF/crusher/e4s-21.08.txt](https://github.com/spack/spack-configs/blob/main/OLCF/crusher/e4s-21.08.txt)
 Articus | ANL | 21.05 | 334 | `gcc@9.3.0` | [https://github.com/spack/spack/tree/e4s-21.05](https://github.com/spack/spack/tree/e4s-21.05) | [https://github.com/spack/spack-configs/blob/main/ANL/JLSE/Arcticus/E4S-21.05/spack.yaml](https://github.com/spack/spack-configs/blob/main/ANL/JLSE/Arcticus/E4S-21.05/spack.yaml) | [https://github.com/spack/spack-configs/blob/main/ANL/JLSE/Arcticus/E4S-21.05/e4s-21.05.txt](https://github.com/spack/spack-configs/blob/main/ANL/JLSE/Arcticus/E4S-21.05/e4s-21.05.txt)
 Articus | ANL | 21.08 | 392 | `gcc@9.3.0`, `oneapi@2021.4.0` | [https://github.com/spack/spack/tree/e4s-21.08](https://github.com/spack/spack/tree/e4s-21.08) | [https://github.com/spack/spack-configs/blob/main/ANL/JLSE/Arcticus/E4S-21.08/prod/spack.yaml](https://github.com/spack/spack-configs/blob/main/ANL/JLSE/Arcticus/E4S-21.08/prod/spack.yaml) | [https://github.com/spack/spack-configs/blob/main/ANL/JLSE/Arcticus/E4S-21.08/e4s-21.08.txt](https://github.com/spack/spack-configs/blob/main/ANL/JLSE/Arcticus/E4S-21.08/e4s-21.08.txt)
-
+Articus | ANL | 21.11 | 426 | `gcc@9.3.0`, `oneapi@2021.4.0` | [https://github.com/spack/spack/tree/e4s-21.11](https://github.com/spack/spack/tree/e4s-21.11) | [https://github.com/spack/spack-configs/blob/main/ANL/JLSE/Arcticus/E4S-21.11/prod/spack.yaml](https://github.com/spack/spack-configs/blob/main/ANL/JLSE/Arcticus/E4S-21.11/prod/spack.yaml) | [https://github.com/spack/spack-configs/blob/main/ANL/JLSE/Arcticus/E4S-21.11/e4s-21.11.txt](https://github.com/spack/spack-configs/blob/main/ANL/JLSE/Arcticus/E4S-21.11/e4s-21.11.txt)
```

Comment:
i just realized the entry names should be `Arcticus` with a `c` between `r` and `t` can you please update this for the other three entries.